src/ArchGDAL.jl

```diff
@@ -38,19 +38,9 @@ include("tables.jl")
 include("geointerface.jl")
 include("convert.jl")
 
-mutable struct DriverManager
-    function DriverManager()
-        drivermanager = new()
-        GDAL.gdalallregister()
-        finalizer((dm,) -> GDAL.gdaldestroydrivermanager(), drivermanager)
-        return drivermanager
-    end
-end
-
-const DRIVER_MANAGER = Ref{DriverManager}()
-
 function __init__()
-    DRIVER_MANAGER[] = DriverManager()
+    GDAL.gdalallregister()
+    atexit(() -> GDAL.cplseterrorhandler(C_NULL))
```

Comment:
I think per @visr's comment, `GDAL.gdaldestroydrivermanager()` should be called in `atexit()`, so this line should be something like 

```
atexit(() -> (GDAL.gdaldestroydrivermanager(); GDAL.cplseterrorhandler(C_NULL)))
```

Reply:
Yes you are right. I'll make the change

Reply:
I guess without the `GDAL.cplseterrorhandler(C_NULL)` part since it is now handled in GDAL.jl